src/composables/multisigAccounts.ts

```diff
@@ -309,12 +311,15 @@ export function useMultisigAccounts({ store, pollOnce = false }: MultisigAccount
   function getMultisigAccountByContractId(contractId: Encoded.ContractAddress) {
     return allMultisigAccounts.value.find((acc) => acc.contractId === contractId);
   }
+
   if (pollOnce && !getStoredMultisigAccounts(nodeNetworkId.value!).length) {
     updateMultisigAccounts();
   } else if (!pollOnce) {
     initPollingWatcher(() => updateMultisigAccounts());
   }
 
+  onNetworkChange(updateMultisigAccounts);
```

Comment:
Little suggestion for future. Generally I think it's a bad pattern to pass the function definition as callback. Why? Consider the fact that the `onNetworkChange` is passing some values to the callback. And in the future someone will update the `updateMultisigAccounts` so it has an boolean argument that decides about something. As the callback value is truthy it will affect the called function in unpredictable way. So in general whenever it's possible I suggest to use arrow functions.